### PR TITLE
ENH bump version for gcc 9.3.0 patch

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin relint

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf This one is ready for review and merge!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "1.24.0.126_gccbb0fb2" %}
+{% set version = "1.24.0.127_g21895fa6ae44c2b6253a0be1994f371dc7f3e8cb" %}
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
{% set version = "1.24.0.130_g21895fa6" %}
```

Comment (beckermr) [reply recipe/meta.yaml]: ahhhhh nice trick there...

**Comment by beckermr** [comment]
merge?

**Pull request merged**